src/server/console/logind_console_services.cpp

```diff
@@ -145,9 +148,9 @@ std::unique_ptr<LogindSeat, decltype(&g_object_unref)> simple_seat_proxy_on_syst
                 using namespace std::literals::string_literals;
                 auto error_msg = error ? error->message : "unknown error";
                 BOOST_THROW_EXCEPTION((
-                    std::runtime_error{
-                        "Failed to connect to DBus interface at "s +
-                        object_path + ": " + error_msg}));
+                                          std::runtime_error{
+                                              "Failed to connect to DBus interface at "s +
+                                              object_path + ": " + error_msg}));
```

Comment:
There are a lot of spurious, whitespace changes to this file. A lot of them away from project style